webdriver-spec.html

```diff
@@ -2049,6 +2049,13 @@
   <a>getting a property</a> named <code>url</code>
   from the <var>parameters</var> argument.
 
+ <li><p>If <var>url</var> is a malformed URL,
```

Comment:
I was on an aeroplane, and the wi-fi stopped working just as I was trying to push my fixup after rebasing. Should be fixed now.